modules/servers/synergywholesale_hosting/synergywholesale_hosting.php

```diff
@@ -340,7 +340,7 @@ function synergywholesale_hosting_synchronize($params)
             'domainstatus' => isset($hostingStatus[$apiResult->status]) ? $hostingStatus[$apiResult->status] : $params['status'],
             'dedicatedip' => $apiResult->dedicatedIPv4,
             'password' => encrypt($apiResult->password),
-            'diskusage'   => $apiResult->disk_usage,
+            'diskusage'   => ($customValues['product'] == SYNERGYWHOLESALE_EMAIL_HOSTING_IDENTIFIER) ? $apiResult->mailboxUsage : $apiResult->disk_usage,
             'disklimit'  => $apiResult->disk_limit,
```

Comment:
Since Disk Limit and the Bandwidth data won't return for Email Hosting Services, can we please not set this data if we're on an email hosting service?